### DISCUSSION TITLE
[Feature Request] live reloading when config file was changed?

### DISCUSSION THREAD

**Discussion by zw963** [xremap/xremap#375] [Ideas]
Consider this is useful feature when try different config, although, not necessary anyway.

thank you.

**Comment by k0kubun** [comment]
I think you're talking about `--watch=config`, which already exists.

**Comment by zw963** [comment]
Hi, i consider we maybe should mention somewhere we can use `comma` to watch devices and config at same times.

as following example:

xremap --watch=device,config /path/to/emacs.yml
